rocAL/rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -167,7 +167,7 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalJpegCOCOFileSourcePartial(RocalConte
                                                             bool is_output,
                                                             bool shuffle = false,
                                                             bool loop = false,
-                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_MAX_SIZE,
+                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_MAX_SIZE_RESTRICTED,
```

Comment:
Can you check if this should be ROCAL_USE_MAX_SIZE_RESTRICTED or user given size restricted?